domain/domain.go

```diff
@@ -218,6 +223,8 @@ func NewDomain(store kv.Storage, lease time.Duration) (d *Domain, err error) {
 	d.ddl = ddl.NewDDL(d.store, d.infoHandle, &ddlCallback{do: d}, lease)
 	d.mustReload()
 
+	variable.RegisterStatistics(d)
```

Comment:
i see in `ddl/ddl.go` have already call `variable.RegisterStatistics(d)`,  so any need to recall here? i am not sure.